content/docs/community/faq.md

```diff
@@ -57,8 +57,6 @@ Cucumber tries to follow the [SemVer](http://semver.org/) specification for rele
 * If the middle number (minor) number in the release changes, you don't need to worry.
 * If the left-hand (major) number changes, you can expect that things might break.
 
-You can read the [history file](https://github.com/cucumber/cucumber/blob/master/History.md) to learn about the changes in every release.
```

Comment:
this can use language style blocks and link to language based implementation changes.